### PR TITLE
vuln(NSWG-ECO-421): angular-http-server

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

**Comment by cjihrig** [comment]
The ID is 421, but the file name is 420.json.

**Comment by lirantal** [comment]
thanks @cjihrig, fixed

File: vuln/npm/421.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "author": "Tung Pun (https://tungpun.pw)",
+  "module_name": "angular-http-server",
+  "cves": [],
+  "vulnerable_versions": "<=1.4.3",
+  "patched_versions": ">=1.4.4",
+  "recommendation": "update angular-http-server to 1.44 or higher",
+  "references": "- https://hackerone.com/reports/330349\n- https://github.com/simonh1000/angular-http-server/commit/8bafc9577161469f5dea01e0b98ea9c525d063e9",
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:N/A:N",
+  "cvss_score": 8.6,
+  "coordinating_vendor": ""
```
Comment (ChALkeR) [inline vuln/npm/421.json]: Should be `null`, empty strings are not allowed.

Comment (lirantal) [reply vuln/npm/421.json]: right. will update the `WgReporter.js` as well

**Pull request merged**